src/components/CardList.js

```diff
@@ -1,25 +1,39 @@
 import React, { useState } from 'react'
+import styled from 'styled-components/macro'
 import Card from './Card'
 import PropTypes from 'prop-types'
 
 CardList.propTypes = {
-  userData: PropTypes.array,
+  profiles: PropTypes.array,
 }
 
-export default function CardList({ userData }) {
+export default function CardList({ profiles }) {
   const [activeCard, setActiveCard] = useState('')
 
   return (
-    <>
-      {userData.map(user => (
-        <Card
-          key={user.id}
-          user={user}
-          {...user}
-          setActiveCard={setActiveCard}
-          isActive={activeCard === user.id}
-        />
-      ))}
-    </>
+    <Scroller>
+      <StyledMain>
+        {profiles.map(user => (
+          <Card
+            key={user.id}
+            user={user}
+            {...user}
```

Comment:
User is passed down as a an object and used that way in `<Card>`, spreading it in again is not necessary, I think.